findmyparadise/src/main/java/com/tp/wrc/findmyparadise/models/Review.java

```diff
@@ -41,13 +41,16 @@
     @Column(name = "joinedDate")
     private LocalDate joinedDate;
 
+    @OneToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
```

Comment:
Are you sure this is a OneToOne relationship? If I am not mistaken then a Reviewer should have a list of reviews, which would make this a ManyToOne.